CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -828,4 +838,17 @@ public static function alterIndexAndTypeForImageURL() {
     return TRUE;
   }
 
+  /**
+   * CRM-18651 Add DataType column to Option Group Table
+   * @return bool
+   */
+  public static function addDataTypeColumnToOptionGroupTable() {
+    if (!CRM_Core_BAO_SchemaHandler::checkIfFieldExists('civicrm_option_group', 'data_type')) {
+      CRM_Core_DAO::executeQuery("ALTER TABLE `civicrm_option_group` ADD COLUMN `data_type` varchar(128) COLLATE utf8_unicode_ci DEFAULT NULL comment 'Data Type of Option Group.'");
```

Comment:
There's an error on upgrade at this point. Screenshot -
![multi_upgrade_error](https://cloud.githubusercontent.com/assets/5929648/18309988/bbd16532-7519-11e6-88b6-b70d84fed631.png)